src/tablespace.h

```diff
@@ -0,0 +1,26 @@
+#ifndef TIMESCALEDB_TABLESPACE_H
+#define TIMESCALEDB_TABLESPACE_H
+
+#include <postgres.h>
+#include "catalog.h"
+
+typedef struct Tablespace
+{
+	FormData_tablespace fd;
+	Oid			tablespace_oid;
+} Tablespace;
+
+typedef struct Tablespaces
```

Comment:
Optional: Out of curiosity why isn't this just a List. If we don't like Lists, we should create a more general data struct for things like this.